channeldb/payment_control.go

```diff
@@ -223,6 +223,19 @@ func (p *PaymentControl) InitPayment(paymentHash lntypes.Hash,
 	return updateErr
 }
 
+// DeleteFailedAttempts deletes all failed htlcs for a payment if configured
+// by the PaymentControl db.
+func (p *PaymentControl) DeleteFailedAttempts(hash lntypes.Hash) error {
+	if !p.db.keepFailedPaymentAttempts {
+		const failedHtlcsOnly = true
```

Comment:
Makes me wish Go had named arguments like Python, but I like the style of introducing some intermediate variables when it makes sense to make the semantics of the passed arguments clearer. It def beats: `foo(1, true, false, true)`.

Reply:
> It def beats: foo(1, true, false, true).

Yeah for sure 😆